doc/start/intro.rst

```diff
@@ -2,42 +2,55 @@
-  the :term:`Ceph Filesystem` (i.e., Ceph Block Devices and Ceph Object Storage
-  do not use MDS). Ceph Metadata Servers make it feasible for POSIX file system 
-  users to execute basic commands like ``ls``, ``find``, etc. without placing 
-  an enormous burden on the Ceph Storage Cluster.
-
-Ceph stores a client's data as objects within storage pools. Using the CRUSH 
-algorithm, Ceph calculates which placement group should contain the object, 
-and further calculates which Ceph OSD Daemon should store the placement group.
-The CRUSH algorithm enables the Ceph Storage Cluster to scale, rebalance, and
-recover dynamically.
-
+Whether you want to provide :term:`Ceph Object Storage` and/or
+:term:`Ceph Block Device` services to :term:`Cloud Platforms`, deploy
+a :term:`Ceph Filesystem` or use Ceph for another purpose, all
+:term:`Ceph Storage Cluster` deployments begin with setting up each
+:term:`Ceph Node`, your network, and the Ceph Storage Cluster. A Ceph
+Storage Cluster requires at least one Ceph Monitor, Ceph Manager, and
+Ceph OSD (Object Storage Daemon). The Ceph Metadata Server is also
+required when running Ceph Filesystem clients.
+
+.. ditaa::  +---------------+ +------------+ +------------+ +---------------+
+            |      OSDs     | | Monitors   | |  Managers  | |      MDSs     |
+            +---------------+ +------------+ +------------+ +---------------+
+
+- **Monitors**: A :term:`Ceph Monitor` maintains maps of the cluster
+  state, including the monitor map, manager map, the OSD map, and the
+  CRUSH map.  This is critical cluster state required for Ceph daemons
+  to coordinate with each other.  Monitors are also responsible for
+  managing authenticate between daemons and clients.  At least three
+  monitors are normally required for redundancy and high availability.
+
+- **Managers**: A :term:`Ceph Manager` daemon is responsible for keeping
+  track of runtime metrics and the current state of the Ceph cluster, including
+  storage utilization, current performance metrics, and system load.  The
+  Ceph Manager daemons also host python-based plugins to manage and expose
+  Ceph cluster information, including a `dashboard`_ and `REST API`_.  At least
+  two managers are normally required for redundancy and high availability.
+
+- **Ceph OSDs**: A :term:`Ceph OSD` (object storage daemon) stores
+  data, handles data replication, recovery, backfilling, rebalancing,
+  and provides some monitoring information to Ceph Monitors and
+  Managers by checking other Ceph OSD Daemons for a heartbeat. At
+  least 3 Ceph OSDs are normally required for redundancy and high
+  availability.
+
+- **MDSs**: A :term:`Ceph Metadata Server` (MDS) stores metadata on
+  behalf of the :term:`Ceph Filesystem` (i.e., Ceph Block Devices and
+  Ceph Object Storage do not use MDS). Ceph Metadata Servers allow
+  POSIX file system users to execute basic commands like ``ls``,
+  ``find``, etc. without placing an enormous burden on the Ceph
```

Comment:
this can be wrapped in `()`:

    basic commands (like `ls` and `find`) without placing [...]

If it must have the *etc* then it needs a comma:

    like ``ls``, ``find``, etc., without placing [...]